tests/integration/ambient/baseline_test.go

```diff
@@ -787,13 +788,81 @@ spec:
   mtls:
     mode: STRICT
   portLevelMtls:
-    8080:
+    80:
```

Comment:
Not sure this is right (nor was 8080). This should be workload port not service port, so 18080? https://github.com/istio/istio/blob/cbb6a2778189d08af99b3076b76fec31e245ea24/pkg/test/framework/components/echo/common/ports/ports.go#L24

Reply:
Ah, I knew there was something off here! Thanks